gnomad_qc/v3/annotations/generate_qc_annotations.py

```diff
@@ -30,6 +27,7 @@ def compute_info() -> hl.Table:
     mt = get_gnomad_v3_mt(key_by_locus_and_alleles=True, remove_hard_filtered_samples=False)
     mt = mt.filter_rows((hl.len(mt.alleles) > 1))
     mt = mt.transmute_entries(**mt.gvcf_info)
+    mt = mt.annotate_rows(alt_alleles_range_array=hl.range(1, hl.len(mt.alleles)))
```

Comment:
Might be worth adding a comment about why this is necessary. It's not obvious that this is used by a function in gnomad_methods. (#109 and broadinstitute/gnomad_methods#258 for context)